### DISCUSSION TITLE
How to get the bazaar price of an item?

### DISCUSSION THREAD

**Discussion by theavgeekbee** [HypixelDev/PublicAPI#356] [Q&A] [answered]
How do I get the price of a bazaar item?

**Comment by shadowtime2000** [comment, marked as answer]
Take [a look at the docs page for Bazaar](https://github.com/HypixelDev/PublicAPI/blob/master/Documentation/methods/skyblock/bazaar.md), it will give you a list of all the items and you can filter through them on your side.

**Reply by theavgeekbee** [reply]
Ok Ill try

**Reply by shadowtime2000** [reply]
Depends on the language you are using, it's pretty simple. Most languages have utilities to do that.
